src/pages/community.tsx

```diff
@@ -222,6 +222,16 @@ function Community() {
       />
       <CommunityChatSection />
       <CommunityMeetingSection />
+      <InfoBanner
+        description="**Searching for Podman Desktop?** Join the [Podman Desktop Community Meetings](https://podman-desktop.io/community#community-events)
```

Comment:
@rujutashinde I like the change overall, I'm just wondering if adding "Community Meetings" here would be better, or does that mess with spacing?

```suggestion
        description="**Searching for Podman Desktop Community Meetings?** Join the [Podman Desktop Community Meetings](https://podman-desktop.io/community#community-events)
```

Reply:
updated
<img width="2550" height="679" alt="image" src="https://github.com/user-attachments/assets/1801b8ba-824c-4e17-9ce8-8aca10e03380" />